### DISCUSSION TITLE
bike's wheel size: radius, diameter or circumference?

### DISCUSSION THREAD

**Discussion by andrearicci** [OpenTracksApp/OpenTracks#604] [General]
Hello,
in the preferences, for bike activity, there's the option to set wheel size, but it's not clear (at least in the italian translation) if it's radius, diameter or circumference. Can you specify?
Thank you

**Comment by dennisguse** [comment]
Thanks for pointing that out!
The english original is "wheel size" and that is not too helpful either.
It is the circumference in mm.

**Reply by andrearicci** [reply]
Thank you!

**Comment by edern-appere** [comment]
Does circumference of the wheel makes changes to the data recorded ? 
I'm only realizing now that I have never set up this before...

**Reply by rgmf** [reply]
> Does circumference of the wheel makes changes to the data recorded ? 

Yes, it does only when a speed sensor is used.

Distance and speed will change according to the circumference when you use speed sensor. So, it's very important setting up that preference.

**Reply by edern-appere** [reply]
Thanks for the information ! 

As of now, I'm only using GPS so, if I understand correctly what you are saying, it shouldn't make any difference on the data OT records on my smartphone.

**Comment by rgmf** [comment]
@edern-appere that's right, if you are only using GPS then don't worry for that preference ;)
